app/code/Magento/Newsletter/Controller/Subscriber.php

```diff
@@ -3,20 +3,29 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
-
-/**
- * Newsletter subscribe controller
- */
 namespace Magento\Newsletter\Controller;
 
 use Magento\Framework\App\Action\Context;
 use Magento\Store\Model\StoreManagerInterface;
 use Magento\Customer\Model\Session;
 use Magento\Newsletter\Model\SubscriberFactory;
 use Magento\Customer\Model\Url as CustomerUrl;
+use Magento\Framework\App\RequestInterface;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+use Magento\Framework\UrlInterface;
+use Magento\Framework\App\ObjectManager;
 
+/**
+ * Newsletter subscribe controller
+ */
 abstract class Subscriber extends \Magento\Framework\App\Action\Action
 {
+    /**
+     * Configuration path to newsletter active setting
+     */
+    const XML_PATH_NEWSLETTER_ACTIVE = 'newsletter/general/active';
```

Comment:
Please add scope to constant. In this case it should be private

Reply:
@VladimirZaets - I have added the scope in constant. Please check.